src/frontend/src/tables/bom/BomTable.tsx

```diff
@@ -347,6 +349,26 @@ export function BomTable({
     table: table
   });
 
+  const validateBom = useApiFormModal({
```

Comment:
Q (non blocking): Is this related to revisions?

Reply:
I added it in as I was creating a number of revision parts (with BOMs) for the demo dataset - https://github.com/inventree/demo-dataset/pull/70